docs/general/web3-and-polkadot.md

```diff
@@ -0,0 +1,196 @@
+IPFS. The Polkadot parachain Crust also provided a similar storage solution.
+
+### Stake Allocation
+
+In Proof-of-Stake blockchains, security is dictated by how much stake is locked on-chain (financial
+security). In a decentralized network, you want to ensure that the difficulty level for a financial
+attack to happen is equal across all nodes.
+{{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }}'s
+[election algorithm](../learn/learn-phragmen.md) makes sure that the stake is maximized across all
+active validators, and the variance in stake across validators is minimized as much as possible.
+
+### Economic Incentives
+
+Strong incentives are essential to avoid spam attacks and incentivize network participants to run
+nodes and secure the network. Strong incentives are possible because blockchain is a trustless
+system where there are no intermediaries between who sends a message and who receives it. Such
+incentives, coupled with punishment for bad behavior, ensure that most of the participants make the
+interest of the network and work together to improve it.
+
+But from where are those incentives coming from? Polkadot's native token
+[DOT is inflationary](../learn/learn-inflation.md). Inflation is used to pay validators for running
+nodes and reward nominators for providing the necessary stake to secure the network. Depending on
+the staking rate, part of the inflation is diverted to the treasury.
+
+### Governance and Treasury
+
+In {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} an on-chain
+[treasury](../learn/learn-polkadot-opengov-treasury.md) together with an
+[open governance](../learn/learn-polkadot-opengov.md) model allow to access funds in a fully
+decentralized manner without any bank transaction whatsoever. This opens up the possibility to come
+to a decision through on-chain voting mechanism, promoting a sense of community and creating an
+independent socio-economical environment.
+
+## Decentralized Access Points
+
+But what if we have data we own stored on a resilient blockchain, but the only way to access the
+blockchain is through an RPC server? Whoever is behind the server or an attacker could present us
+data that is not the truth. We trust the server provider that the data we see is the truth. How can
+we verify that the data are true without trusting anybody?
+
+Here is where light clients play a key role. Light clients are clients that can sit on a web browser
+and can fetch data directly from blockchain full nodes and verify such data using other nodes.
+
+The figure below shows the architectural difference between web2 and web3 applications.
+
+![light-clients](../assets/light-clients.png)
+
+In web2 applications, data are stored on a centralized server, while in web3 applications, data (or
+better data proofs) are stored on the blockchain. With light clients, we can access blockchain data
+through a full node and verify the validity of such data by synching to other nodes. In this way, we
```

Comment:
A light client just connects/syncs with a single full node. There is no verification step involved with syncing to other nodes.

```suggestion
through a full node and verify the validity. In this way, we
```

Reply:
I thought that light clients fetch data from a full node and then verify them because they sync with other nodes (5-10). @bhargavbh I remember you explaining this to me, but I might have misunderstood.

Reply:
Lightclients verify the headers sent by the full node they are connected to. The full node can be connected to multiple nodes, but the light client only has the capacity to connect to a single node, as far as I know.

Reply:
The light client can verify responses because it has the block header (which includes the state-trie merkle root). It gets this block header by warp syncing (an efficient syncing process) with a full node. It is really an implementation detail as to how many full nodes it uses to warp sync, and i would refrain from mentioning it in wiki. Maybe it can be phrased "through a full node and verify the validity of such data. It efficiently syncs (warp sync incase of polkadot) with full node to obtain (Merkle Root) commitment of the latest chain state, and hence can trustlessly verify any response by full node against the commitment". Depending on the target audience, you can trim details in the above statement.